HEN_HOUSE/scripts/bin/egs-parallel

```diff
@@ -0,0 +1,251 @@
+#!/bin/bash
+###############################################################################
+#
+#  EGSnrc script to submit parallel jobs
+#  Copyright (C) 2020 National Research Council Canada
+#
+#  This file is part of EGSnrc.
+#
+#  EGSnrc is free software: you can redistribute it and/or modify it under
+#  the terms of the GNU Affero General Public License as published by the
+#  Free Software Foundation, either version 3 of the License, or (at your
+#  option) any later version.
+#
+#  EGSnrc is distributed in the hope that it will be useful, but WITHOUT ANY
+#  WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+#  FOR A PARTICULAR PURPOSE.  See the GNU Affero General Public License for
+#  more details.
+#
+#  You should have received a copy of the GNU Affero General Public License
+#  along with EGSnrc. If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+#
+#  Author:          Frederic Tessier, 2020
+#
+#  Contributors:
+#
+###############################################################################
+
+
+### help function
+function help {
+    log "HELP"
+    cat <<EOF
+
+    usage:
+
+        $(basename $0) [options] -c | --command 'command'
+
+    options:
+
+        -h | --help         show this help
+        -b | --batch        batch system to use ("cpu" by default)
```

Comment:
It would be great if supported batch systems were listed here. 
Am I misreading the code or can we no longer define some of these as global parameters in the bashrc file?

Reply:
Good idea. Indeed, the `egs-parallel` scripts are completely orthogonal to the legacy `run_user_code_batch` method. Here, the supported batch systems are defined by scripts in `$HEN_HOUSE/scripts/` named `egs-parallel-*`, so if you create a new `egs-parallel-slurm` (ahem, for example 😉 ), them slurm becomes available. It is straightforward to check the scripts folder from `bin/egs-parallel` to list the available options though. Will not do this for this merge, but will work on it afterwards. 👍🏻